src/auro-datepicker.js

```diff
@@ -327,13 +329,14 @@ export class AuroDatePicker extends LitElement {
   }
 
   /**
-   * Changes the calendar's visibility to reflect an updated centralDate.
+   * Changes the calendar's visibility to reflect an updated date.
    * @private
+   * @param {String} date - Date to be displayed in the calendar.
    * @returns {void}
    */
-  handleCentralDateChange() {
-    this.calendar.month = Number(this.centralDate.charAt(1));
-    this.calendar.year = Number(this.centralDate.substring(6));
+  handleVisibleDate(date) {
```

Comment:
I don't think the new function name is semantic with what it is actually doing here.

Also, did the code get added that keeps `centralDate` in sync with `this.month` and `this.year` when those change?

Reply:
Method updated